src/components/Footer/Footer.js

```diff
@@ -232,12 +361,12 @@ const styles = StyleSheet.create({
     paddingRight: 13.5,
     paddingLeft: 18
   },
-  btnComment: {
+  btnComment: (commentValue) => ({
     height: '100%',
     justifyContent: 'center',
-    paddingLeft: 13.5,
-    paddingRight: 10
-  },
+    marginLeft: 0,
+    paddingRight: commentValue > 9 ? 10 : 4
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `btnComment` style has been updated to adjust the paddingRight based on the number of comments. This is a clever way to adjust the style dynamically based on the data.
```diff
- btnComment: (commentValue) => ({
-    height: '100%',
-    justifyContent: 'center',
-   marginLeft: 0,
-   paddingRight: commentValue > 9 ? 10 : 4
- }),
+ btnComment: {
+   height: '100%',
+   justifyContent: 'center',
+  paddingLeft: 13.5,
+  paddingRight: 10
+ },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->